src/org/montclairrobotics/sprocket/control/ArcadeDriveInput.java

```diff
@@ -1,53 +1,60 @@
 package org.montclairrobotics.sprocket.control;
 
-import edu.wpi.first.wpilibj.Joystick;
+import org.montclairrobotics.sprocket.drive.DriveTrainTarget;
 import org.montclairrobotics.sprocket.drive.MotorInputType;
-import org.montclairrobotics.sprocket.drive.DriveTrainInput;
+import org.montclairrobotics.sprocket.geometry.Angle;
 import org.montclairrobotics.sprocket.geometry.Degrees;
 import org.montclairrobotics.sprocket.geometry.Inch;
 import org.montclairrobotics.sprocket.geometry.Polar;
 import org.montclairrobotics.sprocket.geometry.Vector;
+import org.montclairrobotics.sprocket.geometry.XY;
+import org.montclairrobotics.sprocket.pipeline.Step;
+
+import edu.wpi.first.wpilibj.Joystick;
 
-public class ArcadeDriveInput extends DriveTrainInput {
+public class ArcadeDriveInput implements Step<DriveTrainTarget> {
```

Comment:
This change makes joystick inputs indiscernible from other steps in the pipeline (previously I was using DriveTrainInput which stores the direction, rotation, and whether magnitudes were stored as speed or power) and also makes it less straightforward to create other control translators. Me and @rakar (Rich) were having a discussion on Slack earlier about whether that was a necessary distinction. Do you think that this change makes the code base easier to use overall?